usecases/evaluate_scenario/evaluate_scenario.go

```diff
@@ -76,6 +76,12 @@ func EvalScenario(
 	logger.InfoContext(ctx, "Evaluating scenario", "scenarioId", params.Scenario.Id)
 	exec := repositories.ExecutorFactory.NewExecutor()
 
+	// If the scenario has no live version, don't try to Eval() it, return early
```

Comment:
The check was happening too late, causing a panic at line 90 (new line numbers) if a decision was called on the scenario and it was not active.